core/controllers/profile.py

```diff
@@ -148,6 +148,43 @@ def post(self, _):
         self.render_json({})
 
 
+class SubscribeToAndroidList(base.BaseHandler):
+    """Adds user to Android mailing list."""
+
+    EMAIL_REGEX = r'[\w\.\+\-]+\@[\w]+\.[a-z]{2,3}'
```

Comment:
Just to check, where did you get this regex? I think it excludes some valid emails so just checking.

Should there be a ^ and $ indicator so that this doesn't match just a substring? (If this was indeed missed, maybe add some more tests for is_regex_matched or this handler, where appropriate, to validate the behaviour.)

Reply:
For proper email regex see https://emailregex.com/

Reply:
I used it from schema_utils where it is used in the `does_not_contain_email` validator. Yeah, I'll modify this one.

Reply:
Done.